packages/components/src/JSONSchemaRenderer/JSONSchemaRenderer.component.js

```diff
@@ -28,68 +41,109 @@ function InvalidSchemaException() {
 }
 
 /**
- * textRenderer
+ * HiddenRenderer renders nothing to hide specified properties
+ *
+ * @returns null
+ */
+function HiddenRenderer() {
+	return null;
+}
+
+/**
+ * UnkownRenderer renders nothing in case the property doesn't have a
+ * coresponding schema definition
  *
- * @param key
- * @param title
- * @param text
- * @returns {string} - HTML markup for a text component
+ * @returns {undefined}
+ */
+function UnkownRenderer() {
+	return null;
+}
+
+/**
+ * TextRenderer renders text based properties (string and numbers)
  */
-function textRenderer(key, title, text) {
+function TextRenderer({ propertyKey, title, properties }) {
 	return (
-		<div className={classNames('text-renderer', `text-renderer-${key}`)} key={key}>
-			<dt>{title || key}</dt>
-			<dd>{text}</dd>
+		<div className={classNames('text-renderer', `text-renderer-${propertyKey}`)} key={propertyKey}>
+			<dt>{title || propertyKey}</dt>
+			<dd>{properties}</dd>
```

Comment:
if properties is multiple you should iterate over it to render 1 property per `<dd>`